bin/build-all.sh

```diff
@@ -1,25 +1,32 @@
 #!/usr/bin/env bash
+set -e # exiting on error
 
 if [ "$0" == "./bin/build-all.sh" ] && [ -f index.ts ]; then
     echo '**********************************************'
     echo '******* BUILDING ALL TAQUERIA PACKAGES *******'
     echo '**********************************************'
+
     echo ""
     echo '**********************************************'
-    echo "** Installing NPM dependencies"
-    npm install
-    npm run bootstrap
+    echo "** Checking Dependencies"
+    docker ps
```

Comment:
What is this for?

Reply:
Docker is used at the very last steps of `build-all`. So if the engine is not running, the user will be informed only after building everything and failing at the last step. With this change, they will be notified right after the run command `build-all`.